app/javascript/flavours/polyam/styles/components/notification.scss

```diff
@@ -486,12 +485,25 @@
           color: $primary-text-color;
         }
 
+        // Polyam: Upstream only sets color here. Other properties are to match behaviour in status.
         time {
           color: $dark-text-color;
+          text-align: end; // Polyam: Fixes wrapped timestamp leaving gap
```

Comment:
"wrapped" should be "aligned" as wrapping has nothing to do with this.
This is so there is no gap at the end of the string.